ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/AdminNodeInfo.java

```diff
@@ -112,4 +126,39 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
 
     return new JsonRpcSuccessResponse(requestContext.getRequest().getId(), response);
   }
+
+  private URI getNodeAsString(
```

Comment:
Perhaps this should be `getNodeAsURI` or it can remain `getNodeAsString` if you call `toString` here instead of in the calling scope.

Reply:
oops, I add toString